doc/releases/release-notes-4.2.rst

```diff
@@ -181,6 +181,10 @@ New APIs and options
       stack. Most old APIs have been deprecated and the new APIs are used by the Zephyr L2 layer to
       maintain backward compatibility. For migration steps and details, see the
       :ref:`migration guide <migration_4.2>`.
+    * Added the :kconfig:option:`CONFIG_OPENTHREAD_SYS_INIT` Kconfig option to automatically
+      initialize the OpenThread stack during system initialization and
+      :kconfig:option:`CONFIG_OPENTHREAD_SYS_INIT_PRIORITY` to set the priority of the OpenThread
+      system initialization.
```

Comment:
> I also removed redundant text from the release note.

I think you may have forgotten to push the change? Also would be good to clean up the other bullet points as a follow-up / when you have a chance.

```suggestion
    * :kconfig:option:`CONFIG_OPENTHREAD_SYS_INIT`
    * :kconfig:option:`CONFIG_OPENTHREAD_SYS_INIT_PRIORITY`
```

Reply:
I tried to clean up other bullet points. Could you look at the recent changes? Is it better now? Apart from that, I wonder whether I should update the `Deprecated APIs and options` section of the release notes (or the migration guide is enough). What do you think?